src/prefs.ts

```diff
@@ -17,8 +17,16 @@ interface AppWidgets {
     smart_gaps: any,
     snap_to_grid: any,
     window_titles: any,
+    search_engine: any,
 }
 
+const SEARCH_ENGINES: string[] = [
```

Comment:
What if the search engines are based on the first word in the query?

wiki article
bing search
ddg search
google search

Reply:
This makes sense, and a good way to remove it from the settings panel